specification/datalake-analytics/resource-manager/readme.typescript.md

```diff
@@ -0,0 +1,13 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+``` yaml $(typescript)
+typescript:
+  azure-arm: true
+  package-name: "@azure/arm-datalake-analytics"
+  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-datalake-analytics"
```

Comment:
If you look at the original Node.js readme file, you'll see that this was actually going into "azure-arm-datalake" (that's why it has a "source-code-folder-path" property. I'd recommend going back and updating this one by hand.